Mathlib/MeasureTheory/Measure/Content.lean

```diff
@@ -385,19 +403,21 @@ theorem measure_apply {s : Set G} (hs : MeasurableSet s) : μ.measure s = μ.out
   toMeasure_apply _ _ hs
 #align measure_theory.content.measure_apply MeasureTheory.Content.measure_apply
 
+instance outerRegular : μ.measure.OuterRegular := by
+  refine' ⟨fun A hA r (hr : _ < _) => _⟩
```

Comment:
Please use `refine` instead of `refine'` (I know this is not new code from this PR, but since you are moving it around, let's improve it at the same time).